u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java

```diff
@@ -93,8 +94,8 @@ public RegistrationRequest getRegistrationRequest(String accountName, String app
   }
 
   @Override
-  public SecurityKeyData processRegistrationResponse(RegistrationResponse registrationResponse,
-      long currentTimeInMillis) throws U2FException {
+  public SecurityKeyData processRegistrationResponse(
+      RegistrationResponse registrationResponse, long currentTimeInMillis) throws U2FException {
```

Comment:
I looked and couldn't find anything like that in the style guide.  So I went with the default eclipse google style plugin decision.